ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -120,13 +120,20 @@ fi
 
 function upgrade() {
     set -x
-    openshift-tests run-upgrade all \
-        --to-image "${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}" \
-        --options "${TEST_UPGRADE_OPTIONS-}" \
-        --provider "${TEST_PROVIDER}" \
-        -o "${ARTIFACT_DIR}/e2e.log" \
-        --junit-dir "${ARTIFACT_DIR}/junit" &
-    wait "$!"
+    TARGET_RELEASES="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
+    if [[ -f "${SHARED_DIR}/override-upgrade" ]]; then
+        IFS=';' read -ra TARGET_RELEASES < ${SHARED_DIR}/override-releases
+        echo "Overriding upgrade target to ${TARGET_RELEASES[*]}"
+    fi
+    for RELEASE in "${TARGET_RELEASES[@]}"; do
+        openshift-tests run-upgrade all \
+            --to-image "${RELEASE}" \
```

Comment:
Existing jobs like `release-openshift-origin-installer-e2e-aws-upgrade-4.5-to-4.6-to-4.7-to-4.8-ci` use:

```sh
TEST_SUITE=all IMAGE=RELEASE_IMAGES run-upgrade-tests
```

with `RELEASE_IMAGES` replaced by `$next_0,$next_1,$next_2`.  So maybe:

```sh
TARGET_RELEASES="${OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE}"
if [[ -f "${SHARED_DIR}/override-upgrade" ]]; then
    TARGET_RELEASES="$(< "${SHARED_DIR}/override-releases")"
    echo "Overriding upgrade target to ${TARGET_RELEASES[*]}"
fi
openshift-tests run-upgrade all \
  --to-image "${TARGET_RELEASES}" \
  ...
```

after updating origin to use `--to-image` the same way it currently uses `IMAGE`?  Also, you'd need comma delimiters.

Reply:
Ah, `run-upgrade-tests` is just passing `IMAGE` through to `--to-image`, so no need to make any origin updates, it will already work with my suggestion.

Reply:
Oh, `openshift-tests` can already parse comma-delimited images? Nice, experimenting with it in https://github.com/openshift/release/pull/17732

Reply:
Cluster version in https://github.com/openshift/release/pull/17732 shows that comma-delimited release image works, updated the PR.